### PR TITLE
Generic x86_64 support

### PR DESCRIPTION
**Pull request by veldenb**

### CODE REVIEW

File: resources/bin/bootstrap_moonlight-qt.sh [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  echo "Custom audio device: '$ALSA_PCM_NAME'"
+  echo "pcm.!default \"$ALSA_PCM_NAME\"" >> "$CONF_FILE"
+fi
+
+# Stop kodi
+systemctl stop kodi
+
+# Start moonlight-qt and log to log file
+echo "--- Starting Moonlight ---"
+./moonlight-qt "$@"
```
Comment (jhakonen) [inline resources/bin/bootstrap_moonlight-qt.sh:83]: I've seen Moonlight's process left running without its window a few times. Just leaving a black screen with mouse pointer. There doesn't seem to be any way out of it, unless I ssh to LibreELEC and terminate moonlight's process:
```
pkill moonlight-qt
```
And with the `set -e` set, need to also start kodi again:
```
systemctl start kodi
```
It seems to be reproducing on my Linux laptop too, so I made a bug ticket here:
https://github.com/moonlight-stream/moonlight-qt/issues/619

Comment (jhakonen) [reply resources/bin/bootstrap_moonlight-qt.sh:83]: Could perhaps consider adding a trap which would start kodi again if Moonlight's process is killed, or if it crashes. Something like:
```bash
# Stop kodi
systemctl stop kodi

# Start kodi on exit
trap "systemctl start kodi" EXIT

# Start moonlight-qt and log to log file
echo "--- Starting Moonlight ---"
./moonlight-qt "$@"
```

Comment (veldenb) [reply resources/bin/bootstrap_moonlight-qt.sh:83]: I agree, the trap would be better so the user returns to kodi when things start to fall apart for whatever reason :)

Comment (veldenb) [reply resources/bin/bootstrap_moonlight-qt.sh:86]: Fixed in 6b5a608e788701f205270a9c25f697c904af5fa0

**Pull request merged**